### PR TITLE
docs: Add ulimit recommendation to node guide

### PR DESCRIPTION
**Pull request by SilentCicero**
## Description

Until better node utilities are available, ensure node operators increase their ulimit.

### CODE REVIEW

File: docs/guides/docs/running-a-node/running-a-local-node.mdx [hunk 1]
```diff
@@ -19,7 +19,13 @@ There are two types of Fuel networks that can be run:
 
 An in-memory node does not persist the blockchain state anywhere, it is only stored in memory as long as the node is active and running.
 
-To spin-up a local in-memory Fuel node download or copy the local snapshot from [here](https://github.com/FuelLabs/chain-configuration/tree/master/local), then run the following command:
+First ensure your environments [open files limit](https://askubuntu.com/questions/162229/how-do-i-increase-the-open-files-limit-for-a-non-root-user) `ulimit` is increased, example:
+
+```sh
+ulimit -S -n 4096
```
Comment (xgreenx) [inline docs/guides/docs/running-a-node/running-a-local-node.mdx:25]: I would suggest `ulimit -S -n 32768`

Comment (SilentCicero) [reply docs/guides/docs/running-a-node/running-a-local-node.mdx:25]: Will adjust @xgreenx

Comment (SilentCicero) [reply docs/guides/docs/running-a-node/running-a-local-node.mdx]: @xgreenx adjusted to:
```ulimit -S -n 32768```

**Comment by JoshuaBatty** [comment]
Just looking into why the link check CI job is failing. 

Looks like this link was updated to the correct URL 2 weeks ago [here](https://github.com/FuelLabs/sway/pull/6064/files#diff-d83081a0e3ee4a04c57dd850d9ae36f194ad2e8763e6e7530c1870eb6e3bd544R77) 

This fix was included in the `sway 0.61.0` release but the docs hub seems to still be pointing to the `0.60` release. I'm not familiar with how devrel pull in the latest docs changes. It looks like the sway repo submodule needs to be updated which Sarah used to do. Not sure who we should ping here to do this?

**Review by calldelegation — APPROVED** [review]

**Pull request merged**